governance/votes/Executive vote - March 22, 2021.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Template - [Executive Vote] Approve March 2021 Governance Cycle Bundle - March 22, 2021
+
+---
+# [Executive Proposal] Approve March 2021 Governance Cycle Bundle - March 22, 2021
```

Comment:
Can you restructure first section and executive summary to match the format in the template? Pablo and I put a fair bit of thought into the optimal ordering of information. 

Title
1. How did this vote come about
2. Resources on  how to vote if you are new.

Executive Summary:
1. Contents
2. GSM Pause Delay
3. Office Hours (if present)
4. Expiry

Reply:
Ok I added the GSM Pause Delay, but I think it's irrelevant for this vote since there are no on-chain changes.

Same goes for office hours.

Reply:
in my edit I put expiry above the office hours and gsm since it's more important/higher priority information

Reply:
Fair points, by all means leave GSM and Office Hours parts out.

Reply:
Was more referring to the other re-ordering.